src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/proxy/Proxy.java

```diff
@@ -89,6 +89,19 @@
     private static final String SSLCONTEXT_PROTOCOL =
         JMeterUtils.getPropDefault("proxy.ssl.protocol", "TLS"); // $NON-NLS-1$ $NON-NLS-2$
 
+    private static final String PROTOCOL_LIST =
+            JMeterUtils.getPropDefault("https.socket.protocols", ""); // $NON-NLS-1$ $NON-NLS-2$
+
+    private static final String CIPHER_SUITE_LIST =
+            JMeterUtils.getPropDefault("https.cipherSuites", ""); // $NON-NLS-1$ $NON-NLS-2$
+
+    private static final String[] SUPPORTED_PROTOCOL_LIST =
+            PROTOCOL_LIST.isEmpty() ?
+                    null: PROTOCOL_LIST.split(" "); // $NON-NLS-1$
```

Comment:
What do you think if trim the value first, and then split with `\\s+` pattern? In other words, allow multiple whitespace as a separator